operations/list.go

```diff
@@ -20,13 +20,14 @@ import (
 
 func List() cli.Command {
 	const (
-		projectsFlagName   = "projects"
-		variantsFlagName   = "variants"
-		tasksFlagName      = "tasks"
-		distrosFlagName    = "distros"
-		spawnableFlagName  = "spawnable"
-		parametersFlagName = "parameters"
-		aliasesFlagName    = "aliases"
+		projectsFlagName       = "projects"
+		variantsFlagName       = "variants"
+		tasksFlagName          = "tasks"
+		distrosFlagName        = "distros"
+		spawnableFlagName      = "spawnable"
+		parametersFlagName     = "parameters"
+		patchAliasesFlagName   = "patch-aliases"
```

Comment:
I would argue against changing the name of an existing command, since it may already be part of people's workflows.

Reply:
Thanks! I did this instead. 
![](https://files.slack.com/files-pri/T01C4Q4H3CL-F01LQCEHNTT/image.png)

Reply:
I like this approach! You'll want to make a new ticket to remove the other alias in a month or so.